custom/icds_reports/ucr/data_sources/dashboard/remove_member_form.json

```diff
@@ -0,0 +1,103 @@
+{
+  "domains": [
+    "icds-dashboard-qa",
+    "sankalp_cas",
+    "zohaib-sandbox",
+    "akshita-sandbox",
+    "sunaina-sandbox",
+    "laurence-project-1",
+    "jessica-icds-cas",
+    "marissa-test",
+    "derek-icds-sandbox",
+    "priyanka-app",
+    "shrena-dev",
+    "aparatest",
+    "reach-sandbox",
+    "reach-dashboard-qa",
+    "reach-test",
+    "icds-cas",
+    "cas-lab"
+  ],
+  "server_environment": [
+    "india",
+    "icds"
+  ],
+  "config": {
+    "table_id": "static-remove_member",
+    "display_name": "Forms - Remove Member (REACH Dashboard)",
```

Comment:
this is not REACH dashboard. its ICDS-CAS dashboard

Reply:
changed